src/routes/meteorsData.js

```diff
@@ -1,10 +1,13 @@
 const express = require('express');
 const router = express.Router();
 const meteorsService = require('../services/meteorsData');
+const MeteorRequest = require('../Requests/meteorRequest');
 
 router.get('/', async function (req, res, next) {
+  const request = new MeteorRequest(req.query);
```

Comment:
Actually I'd stay with this entity - it checks if parameters are presented in request and parse values - I think it would be better to avoid parsing query in service

Reply:
Renamed to dto